### ISSUE TITLE
Auth doesn't work

### ISSUE THREAD

**Issue by syabro** [pottava/aws-s3-proxy#49]
I've added `BASIC_AUTH_USER` and `BASIC_AUTH_PASS` but when I go to `http://localhost:8000` it doesn't ask for credentials - just show the directory listing.

My docker-compose

```
proxy:
  image: pottava/s3-proxy
  restart: always
  ports:
    - 8000:80
  environment:
    - AWS_REGION=us-east-1
    - AWS_ACCESS_KEY_ID=
    - AWS_SECRET_ACCESS_KEY=
    - AWS_S3_BUCKET=
    - ACCESS_LOG=true
    - DIRECTORY_LISTINGS=true
    - DIRECTORY_LISTINGS_FORMAT=html
    - BASIC_AUTH_USER=admin
    - BASIC_AUTH_PASS=pass
  container_name: proxy
``

**Comment by unfor19** [comment]
Yup, same here, doesn't ask for credentials when providing BASIC_AUTH_USER and BASIC_AUTH_PASSWORD

Here are my logs-

![s3-proxy-auth](https://user-images.githubusercontent.com/15122452/91472935-d7bb2900-e8a0-11ea-8874-961990dac411.jpg)

**Comment by dev-d** [comment]
Issue seems to be fixed in version 1, but broken in version 2.0.0 and 2.0.1

**Comment by patrickdk77** [comment]
Do you want to try my fork? I tried to find why it wasn't working, but for me it works. Not sure what change I did that might have fixed it.
